src/js/builtins/ImportMetaObject.ts

```diff
@@ -109,7 +109,7 @@ export function requireESM(this: ImportMetaObject, resolved) {
   return exports;
 }
 
-$visibility = "Private";
+$visibility = "PrivateRecursive";
 export function internalRequire(this: ImportMetaObject, id) {
   var cached = $requireMap.$get(id);
```

Comment:
Depending on how much these restrict the stack we may not want to do this. It's mostly trying them on.